web/firefoxcom.js

```diff
@@ -345,6 +345,11 @@ class FirefoxExternalServices extends DefaultExternalServices {
     );
     return shadow(this, "supportedMouseWheelZoomModifierKeys", support);
   }
+
+  static get isInAutomation() {
+    const isInAutomation = FirefoxCom.requestSync("isInAutomation");
```

Comment:
Since I had to look up what `isInAutomation` did exactly, would it be good to add a one-line comment here along the lines of "`isInAutomation` checks if the code is run in the context of the Firefox test suite"?